.github/workflows/elixir.yml

```
@@ -3,56 +3,29 @@ name: Elixir CI
 on: [push, pull_request]
 
 jobs:
-  build_with_opt_24:
-    name: Build and test older versions ${{matrix.elixir}}-otp-${{matrix.otp}}
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        otp: ['24.0']
-        elixir: ['1.12.0', '1.13.0', '1.14.0']
-    steps:
-    - uses: actions/checkout@v2
-    - name: Set up Elixir
-      uses: erlef/setup-elixir@v1
-      with:
-        otp-version: ${{matrix.otp}}
-        elixir-version: ${{matrix.elixir}}
-    - name: Restore dependencies cache
-      uses: actions/cache@v2
-      with:
-        path: deps
-        key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
-        restore-keys: ${{ runner.os }}-mix-
-    - name: Install dependencies
-      run: mix deps.get
-    - name: Run tests
-      run: mix test
-    - name: format and lint
-      run: mix lint.all
-
-  build_with_opt_25:
+  build:
     name: Build and test ${{matrix.elixir}}-otp-${{matrix.otp}}
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        otp: ['25.0']
-        elixir: ['1.13.0', '1.14.0']
+        otp: ["24.0", "25.0"]
+        elixir: ["1.13.0", "1.14.0"]
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Elixir
-      uses: erlef/setup-elixir@v1
-      with:
-        otp-version: ${{matrix.otp}}
-        elixir-version: ${{matrix.elixir}}
-    - name: Restore dependencies cache
-      uses: actions/cache@v2
-      with:
-        path: deps
-        key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
-        restore-keys: ${{ runner.os }}-mix-
-    - name: Install dependencies
-      run: mix deps.get
-    - name: Run tests
-      run: mix test
-    - name: format and lint
-      run: mix lint.all
+      - uses: actions/checkout@v2
+      - name: Set up Elixir
+        uses: erlef/setup-elixir@v1
+        with:
+          otp-version: ${{matrix.otp}}
+          elixir-version: ${{matrix.elixir}}
+      - name: Restore dependencies cache
+        uses: actions/cache@v2
+        with:
+          path: deps
+          key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
+          restore-keys: ${{ runner.os }}-mix-
+      - name: Install dependencies
+        run: mix deps.get
+      - name: format and lint
+        run: mix lint.all
+      - name: Run tests
+        run: mix coveralls.github --warnings-as-errors
```

If no plans to use https://coveralls.io/ we can apply this suggestion
```suggestion
        run: mix test --warnings-as-errors
```